distribution/std-lib/Base/src/Data/Time/Date.enso

```diff
@@ -11,7 +11,7 @@ polyglot java import org.enso.base.Time_Utils
 
 type Date
 
-    ## `Date` represents a date, often viewed as year-month-day. For example,
```

Comment:
This change was introduced because somehow starting a text with inline broke indentation on the next line, tried fixing it without success.